### PR TITLE
Added sample typing for WordcloudTransform from Vega

### PR DESCRIPTION
**Pull request by hungngo97**
There are a few questions I am not sure while implementing WordcloudTransform:
1. Vega.schema is used to create node but when is transform typing used?
2. How would I be able to test new vega typing?

I will definitely redo this typing and add comments for it, I just want to make sure that I get the idea right. Thanks

### CODE REVIEW

File: src/vega.schema.ts [hunk 1]
```diff
@@ -419,6 +420,22 @@ export interface VgGraticuleTransform {
   precision?: number;
 }
 
+export interface VgWordcloudTransform {
```
Comment (domoritz) [inline src/vega.schema.ts]: Can you move this to vega?

**Comment by domoritz** [comment]
@hungngo97 are you still working on this?

**Comment by hungngovinh** [comment]
Yea but let me finish up the gradient typing pr though

**Comment by domoritz** [comment]
I'm going to close this PR. Please open a new one.

**Pull request closed without merging**